packages/create-styles/src/components/ThemeProvider/ThemeProvider.utils.js

```diff
@@ -199,8 +203,10 @@ export function useThemeStyles({
 		 * We only want to update + set the theme if there's a change.
 		 * Since themes (potentially) be nested, we need to do a shallowEqual check.
 		 */
+		// @ts-ignore
 		if (shallowEqual(themeRef.current, theme)) return;
 
+		// @ts-ignore
 		themeRef.current = theme;
```

Comment:
Not allowed to assign to `current` as it is `readonly` 🤔 Is this something we just ignore when writing libraries?

Reply:
There's a note about this here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/77d6faf16f14832e88c28aa34bf86e3a67520a69/types/react/index.d.ts#L1029-L1043

We'll want a `MutableRefObject` [above](https://github.com/ItsJonQ/g2/pull/104/files/e487d30475a9d141405ddfae473b949cd497693a#diff-b132a18aa8ec54f906c1b8484058cd5d3f636def3bdfe93ac37a58bc16b511efR176).